matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/crypto/keysbackup/KeysBackup.kt

```diff
@@ -402,9 +402,8 @@ internal class KeysBackup @Inject constructor(
                 deviceId = components[1]
             }
 
-            var device: MXDeviceInfo? = null
             if (deviceId != null) {
-                device = cryptoStore.getUserDevice(deviceId, myUserId)
+                var device = cryptoStore.getUserDevice(deviceId, myUserId)
```

Comment:
val?

there is also a remaining !! at line 414 device.fingerprint()!!

Reply:
done